build.sbt

```diff
@@ -55,17 +55,20 @@ ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value
 
 libraryDependencies ++= Seq(
   "com.metamx" %% "scala-util" % "1.11.3" exclude("log4j", "log4j") force(),
-  "com.ircclouds.irc" % "irc-api" % "1.0-0014"
+  "com.ircclouds.irc" % "irc-api" % "1.0-0014",
+  "org.apache.kafka" % "kafka-clients" % "0.9.0.0",
+  "ch.qos.logback" % "logback-core" % "1.1.2",
```

Comment:
The original reason didn't have anything to do with minimizing log output on the console, that could be done with a logback config that routes logs to a file or just sets the level to OFF. It was really just about the app/library thing.